src/main/resources/database/changes/release-10.37.0/casesvc_bres_foundation_schema.sql

```diff
@@ -48,6 +48,7 @@ CREATE TABLE "case" (
     collectioninstrumentid character varying(128),
     state character varying(20),
     actionplanid integer,
+    active_enrolment boolean,
```

Comment:
You can't do this.

The database is managed by liquibase, so you will need to do an alter table statement in a new release

Reply:
Fixed.